packages/admin-panel/src/VizBuilderApp/components/MapOverlay/MapOverlayMetadataForm.jsx

```diff
@@ -131,6 +133,17 @@ export const MapOverlayMetadataForm = ({ Header, Body, Footer, onSubmit }) => {
           }}
           onChange={(thing, selected) => setCountryCodes(selected)}
         />
+
+        <JsonEditorInputField
+          id="entityAttributesFilter"
+          name="entityAttributesFilter"
+          inputKey="entityAttributesFilter"
+          label="Entity Attributes Filter"
+          onChange={(_, value) => setEntityAttributesFilter(value)}
+          value={entityAttributesFilter}
+          stringify={false}
+          helperText='This field will be used to filter the entities that this map overlay will have data for. This field is case sensitive. It is an extension of `config.measureLevel`. E.g. {"facility_type": "Hospital"}'
+        />
```

Comment:
Ah nice, I hadn't considered how we ought to add this here. I think I'd also be okay with us defaulting to empty object, and having people come back later and configure it if they need? Feels a bit advanced to be shown in every scenario